src/views/DocumentationTopic.vue

```diff
@@ -332,3 +332,12 @@ export default {
   },
 };
 </script>
+<style lang="scss" scoped>
+@import 'docc-render/styles/_core.scss';
+
+.nav-title {
+  font-size: inherit;
```

Comment:
It seems like the `font-size` and `font-weight` you're adding here just get overwritten—do we really need these rules?

Reply:
We need them for the h2 in the navigator, I moved them there.
This is because if we don't set the `h2` tag to be `inherit` it gets the `h2` styles by default.

The overwriting of the styles only happens in the `NavBase` which it's a different component